static/app/components/modals/inviteMembersModal/inviteMembersContext.tsx

```diff
@@ -0,0 +1,26 @@
+import {createContext} from 'react';
+
+import type {
+  InviteRow,
+  InviteStatus,
+  NormalizedInvite,
+} from 'sentry/components/modals/inviteMembersModal/types';
+
+export type InviteMembersContextProps = {
+  complete: boolean;
+  inviteStatus: InviteStatus;
+  invites: NormalizedInvite[];
+  pendingInvites: InviteRow[];
+  reset: () => void;
+  sendInvites: () => void;
+  sendingInvites: boolean;
+  setEmails: (emails: string[], index: number) => void;
+  setRole: (role: string, index: number) => void;
+  setTeams: (teams: string[], index: number) => void;
+  willInvite: boolean;
+  error?: string;
+};
+
+export const InviteMembersContext = createContext<InviteMembersContextProps | undefined>(
+  undefined
+);
```

Comment:
Rather than using `undefined` here, it's more common practice to provide a skeleton / loading state of the context - where things are either in "pending" or "off" states.
```suggestion
export const InviteMembersContext = createContext<InviteMembersContextProps>({
  complete: false,
  inviteStatus: {},
  invites: [],
  pendingInvites: [],
  reset: () => {},
  sendInvites: () => {},
  sendingInvites: false,
  setEmails: () => {},
  setRole: () => {},
  setTeams: () => {},
  willInvite: false,
});
```

This ends up helping with the type definitions as well - it helps point out if things are optional or not and what the initial states should look like.